build.gradle

```diff
@@ -1,20 +1,27 @@
+import io.gitlab.arturbosch.detekt.Detekt
+
 buildscript {
     ext.versions = [
-            kotlin: '1.5.20',
-            minSdk: 21,
-            compileSdk: 30,
-            androidTools: '27.2.2',
+            kotlin       : '1.7.20',
+            minSdk       : 23,
+            compileSdk   : 33,
+            androidTools : '30.4.0-rc03',
+            detekt       : '1.22.0',
+            gradle       : '7.3.1',
```

Comment:
This is pretty misleading. This is not the Gradle version, but the Android Gradle Plugin version